public/locales_uid2/en.json

```diff
@@ -17,25 +17,26 @@
   "Powered by": "Powered by",
   "Internal Server Error": "Internal Server Error",
   "EXIT": "EXIT",
-  "Welcome!": "Welcome!",
-  "trusted-partners-committed": "Our trusted advertising partners using the UID identifier are committed to delivering relevant advertising that also respects your choice.",
-  "Advertising-enables-access": "Advertising enables access to your favorite content across the internet.",
+  "Welcome!": "Hello,",
```

Comment:
Was unsure if I should change the key for this, given I would have to change the japanese one without knowing the proper translation. Just figured I would leave as is.